test/parallel/test-tick-processor.js

```diff
@@ -57,4 +60,5 @@ function runTest(pattern, code) {
                         {encoding: 'utf8'});
   assert(pattern.test(out));
   fs.unlinkSync(log);
-}
+  fs.unlinkSync(testCodePath);
+}
```

Comment:
Missing line feed at the end of the file should result in a lint error if you run `make jslint` or `make test`.